src/libsync/creds/httpcredentials.cpp

```diff
@@ -218,7 +218,7 @@ void HttpCredentials::fetchFromKeychainHelper()
             return;
         }
         const auto data = job->data().toString();
-        if (OC_ENSURE(!data.isEmpty())) {
+        if (OC_ENSURE((!data.isEmpty()))) {
```

Comment:
Why this? if something is wrong with `OC_ENSURE`, shouldn't you fix that? Or is fixing that too risky?

Reply:
Committed by accident, well spotted.